src/panels/config/automation/action/types/ha-automation-action-wait_for_trigger.ts

```diff
@@ -43,12 +43,12 @@ export class HaWaitForTriggerAction
         )}
       >
         <ha-switch
-          .checked=${continue_on_timeout ?? true}
+          .checked=${this.action.continue_on_timeout ?? true}
           @change=${this._continueChanged}
         ></ha-switch>
       </ha-formfield>
       <ha-automation-trigger
-        .triggers=${wait_for_trigger}
+        .triggers=${ensureArray(this.action.wait_for_trigger)}
```

Comment:
@frenck please review this adjustment to #13426 which might be a bug?  The type you created accepts `Trigger | Trigger[]`, but the automation component needs an array.